README.md

```diff
@@ -1,118 +1,171 @@
+**Note:** The built-in CLI server is *for development only*.
 
-If you don't have composer installed globally then another way to create a new ZF2 project is to download the tarball and install it:
+## Using Vagrant
 
-1. Download the [tarball](https://github.com/zendframework/ZendSkeletonApplication/tarball/master), extract it and then install the dependencies with a locally installed Composer:
+This skeleton includes a `Vagrantfile` based on ubuntu 14.04, and using the
+ondrej/php PPA to provide PHP 7.0. Start it up using:
 
-        cd my/project/dir
-        curl -#L https://github.com/zendframework/ZendSkeletonApplication/tarball/master | tar xz --strip-components=1
-    
+```bash
+$ vagrant up
+```
 
-2. Download composer into your project directory and install the dependencies:
+Once built, you can also run composer within the box. For example, the following
+will install dependencies:
 
-        curl -s https://getcomposer.org/installer | php
-        php composer.phar install
+```bash
+$ vagrant ssh -c 'cd /var/www ; composer install'
+```
 
-If you don't have access to curl, then install Composer into your project as per the [documentation](https://getcomposer.org/doc/00-intro.md).
+While this will update them:
 
-Web server setup
-----------------
+```bash
+$ vagrant ssh -c 'cd /var/www ; composer update'
+```
 
-### PHP CLI server
+While running, Vagrant maps your host port 8080 to port 80 on the virtual
+machine; you can visit the site at http://localhost:8080/
 
-The simplest way to get started if you are using PHP 5.4 or above is to start the internal PHP cli-server in the root
-directory:
+## Using Docker
 
-    php -S 0.0.0.0:8080 -t public/ public/index.php
+This skeleton provides a `Dockerfile` for use with Docker. Build the image
+using:
 
-This will start the cli-server on port 8080, and bind it to all network
-interfaces.
+```bash
+$ docker build -t <name> .
+```
 
-**Note:** The built-in CLI server is *for development only*.
+Where `<name>` is a unique name for the image on your machine (this will usually
+be your project name). Once built, you can run it. To do so, you should map a
+port on your host machine to the image's port 80, and map the current directory
+to `/var/www` on the image:
+
+```bash
+$ docker run -d -p 8080:80 -v $(pwd):/var/www <name>
+```
+
+At this point, you can visit http://localhost:8080 to see the site running.
+
+You can also run composer from the image. You will need to map your current
+directory to `/var/www` on the image when doing so:
 
-### Vagrant server
+```bash
+$ docker run -v $(pwd):/var/www <name> composer install
+```
 
-This project supports a basic [Vagrant](http://docs.vagrantup.com/v2/getting-started/index.html) configuration with an inline shell provisioner to run the Skeleton Application in a [VirtualBox](https://www.virtualbox.org/wiki/Downloads).
+Remembering to map the volume and port when running vanilla Docker can become
```

Comment:
Is docker-compose installed by default when installing docker now? If so, I can remove those instructions; if not, it's probably good to have vanilla docker instructions just in case.